packages/composables/src/composables/useCurrency.ts

```diff
@@ -1,12 +1,12 @@
 import { Ref, watch, getCurrentInstance, computed, ref, nextTick } from 'vue'
 import { FzUseCurrencyOptions } from '../types'
-import { format as formatNumber, parse } from '../utils'
+import { format as formatNumber, parse, roundTo } from '../utils'
 
 export const useCurrency = (options: FzUseCurrencyOptions) => {
   const inputRef: Ref<HTMLInputElement | null | undefined> = ref(null)
   const vm = getCurrentInstance()
```

Comment:
Accessing the current instance has made this composable highly coupled with the `FzCurrencyInput`: components using this composable must have an `amount` and a `nullOnEmpty` prop, as well as an emitter for the `update:amount` event.  

For now, this is not a major issue, but we should probably create a refactor card to move these composables into the input package.

Reply:
Correct. The reason I made it this way is I loosely followed what they did in the existing components we used as examples. 
I kind of understnad why it was done this way, it's quite hard to decouple it from components while maintaining reactivity

Reply:
although I would argue this specific PR has little to do with it. If anything, I added utils (which might be a better solution, along with a refactor that moves component logic... well into the component itself I guess)